lib/modules/apostrophe-ui/public/css/components/tooltip.less

```diff
@@ -0,0 +1,35 @@
+.apos-tooltip
+{
```

Comment:
I don't see tooltips.less on the "changed" page for this patch... did you add an import somewhere?